### PR TITLE
db config

### PR DESCRIPTION
**Pull request by martachupil**

### CODE REVIEW

File: server/routes/users.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    try {
+      const u = User.findByPk(req.payload.id);
+      res.status(200).json(u);
+    } catch (err) {
+      res.status(404).json({message: err});
+    }
+});
+
+router.post('/register', (req, res) => {
+  const {name, email, password } = req.body;
```
Comment (sgsasha) [inline server/routes/users.js]: This code does not work :( 

But I see stuff that might work in authentication.module. I believe you should import that module and call `register()` from that module

File: server/models/user/index.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+const Sequelize = require('sequelize');
+const crypto = require('crypto');
+var jwt = require('jsonwebtoken');
+const db = require('../db');
+
+let secret = null;
+
+class User extends Sequelize.Model {
+    set password(value) {
```
Comment (sgsasha) [inline server/models/user/index.js]: `password` => `setPassword`. On my opinion `setPassword` is more descriptive.

File: server/models/user/index.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        const salt = crypto.randomBytes(16).toString('hex');
+        const hash = crypto.pbkdf2Sync(value, salt, 1000, 64, 'sha512').toString('hex');
+        this.setDataValue('salt', salt);
+        this.setDataValue('hash', hash);
+    }
+
+    validPassword(pass) {
+        const hash = crypto.pbkdf2Sync(password, this.salt, 1000, 64, 'sha512').toString('hex');
```
Comment (sgsasha) [inline server/models/user/index.js]: You should compare your `const hash` with `this.hash`. Basically you are generating new hash with entered password and compare with existing hash in db.

File: server/routes/users.js (hunk 1 above)
Comment (sgsasha) [reply server/routes/users.js]: Same for login :)

File: server/models/user/index.js (hunk 2 above)
Comment (martachupil) [reply server/models/user/index.js]: @sgsasha this is **setter**

File: server/models/user/index.js (hunk 3 above)
Comment (martachupil) [reply server/models/user/index.js]: @sgsasha fixed

File: server/routes/users.js (hunk 1 above)
Comment (martachupil) [reply server/routes/users.js]: @sgsasha fixed

**Pull request merged**